base/reflection.jl

```diff
@@ -84,7 +84,7 @@ are also included. Names are returned in sorted order.
 As a special case, all names defined in `Main` are considered \"public\",
 since it is not idiomatic to explicitly mark names from `Main` as public.
 
-See also: [`isexported`](@ref), [`ispublic`](@ref), [`@locals`](@ref Base.@locals), [`@__MODULE__`](@ref).
+See also: [`Base.isexported`](@ref), [`Base.ispublic`](@ref), [`Base.@locals`](@ref Base.@locals), [`@__MODULE__`](@ref).
```

Comment:
Maybe like this (I am not sure what the `@ref Base.@locals` does, so I might be guessing wrong)
```suggestion
See also: [`Base.isexported`](@ref), [`Base.ispublic`](@ref), [`Base.@locals`](@ref), [`@__MODULE__`](@ref).
```